docs/prod-host-setup.md

```diff
@@ -317,6 +317,15 @@ echo "KSM: ENABLED (Recommendation: DISABLED)"
 
 #### Check for mitigations against Spectre Side Channels
 
+In development we use an integration test to check for spectre vulnerability on
+the host.
+
+The script we run in this test can be downloaded and executed like:
+
+```bash
+wget -O - https://meltdown.ovh | bash
```

Comment:
This idiom always gives me unease, as it's liable to MITM attacks https://0x46.net/thoughts/2019/04/27/piping-curl-to-shell/. Can we have it as two steps?

```
wget > script
# Check the script 
bash script
```

or, highlight the risk here?

Reply:
I share this concern, although I think in practicality people treat:
```bash
wget -O - https://meltdown.ovh | bash
```
the same as if it where 2 steps.

I've added a comment.